### PR TITLE
ref(quotas): Remove unused get_maximum_quota

### PR DESCRIPTION
**Pull request by jan-auer**
Removes `quotas.get_maximum_quota` which has been unused for a while.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx [hunk 1]
```diff
@@ -135,3 +112,9 @@ export default class OrganizationRateLimit extends React.Component {
     );
   }
 }
+
+const Box = styled('div')`
+  display: flex;
+  flex: 1;
+  padding: 0 2px;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx]:
```suggestion
  padding: 0 ${space(2)};
```

File: src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from '@emotion/styled';
 
 import {t, tct} from 'app/locale';
-import Field from 'app/views/settings/components/forms/field';
 import Form from 'app/views/settings/components/forms/form';
 import {Panel, PanelAlert, PanelBody, PanelHeader} from 'app/components/panels';
 import RangeField from 'app/views/settings/components/forms/rangeField';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx]:
```suggestion
import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
import space from 'app/styles/space';
```

File: src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx [hunk 3]
```diff
@@ -72,43 +68,24 @@ export default class OrganizationRateLimit extends React.Component {
...
+              <RangeField
+                name="accountRateLimit"
+                label={t('Account Limit')}
+                min={0}
+                max={1000000}
+                allowedValues={ACCOUNT_RATE_LIMIT_VALUES}
+                help={t(
+                  'The maximum number of events to accept across this entire organization.'
+                )}
+                placeholder="e.g. 500"
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/settings/organizationRateLimits/organizationRateLimits.jsx]:
```suggestion
                placeholder={t('e.g. 500')}
```

**Review by matejminar — APPROVED** [review]
Thank you for removing reflexbox 👏

File: tests/sentry/api/endpoints/test_organization_details.py [hunk 4]
```diff
@@ -70,7 +70,7 @@ def test_with_projects(self):
 
         # TODO(dcramer): we need to pare this down -- lots of duplicate queries
         # for membership data
-        with self.assertNumQueries(36, using="default"):
+        with self.assertNumQueries(35, using="default"):
```
Comment (jan-auer) [inline tests/sentry/api/endpoints/test_organization_details.py]: We removed the check for the option, which is why there's one less query.

**Comment by jan-auer** [comment]
Turns out this is not actually dead, but instead rendered in some special cases. Let's leave it in.

**Pull request closed without merging**